crates/resolver/src/resolver.rs

```diff
@@ -1092,174 +1064,146 @@ mod tests {
         assert!(is_send_t::<LookupFuture<GenericConnection>>());
     }
 
-    #[test]
-    fn test_lookup_google() {
-        use super::testing::lookup_test;
-        let io_loop = Runtime::new().expect("failed to create tokio runtime");
+    #[tokio::test]
+    async fn test_lookup_google() {
+        subscribe();
         let handle = TokioConnectionProvider::default();
-        lookup_test::<Runtime, TokioConnectionProvider>(ResolverConfig::google(), io_loop, handle)
+        lookup_test::<TokioConnectionProvider>(ResolverConfig::google(), handle).await;
```

Comment:
I think the connection provider can be inferred in this and other tests in this module? (As in, no need for `::<TokioConnectionProvider>`.)